docs/troubleshooting.md

```diff
@@ -139,6 +140,21 @@ kubectl delete secret pipelines-as-code-secret -n pipelines-as-code
 6. :gear: On the PR page, type `/retest` on the comment box and post the comment.
    Observe the behavior once more.
 
+# PR fails when Webhook Secret was not added
+
+When creating a github application if webhook secret was not added we get following error in our opened PR:
```

Comment:
```quote
 we get following error in our opened PR
```
Let's be more specific about where we see it. In the Konflux pipeline logs? In the status that is reported to the PR?